e2e/test-app-v9/angular.json

```diff
@@ -18,6 +18,7 @@
             "main": "src/main.ts",
             "polyfills": "src/polyfills.ts",
             "tsConfig": "tsconfig.app.json",
+            "aot": true,
```

Comment:
Put `aot: true` here to be inline with new configuration starting from v9, see https://angular.io/guide/ivy#aot-and-ivy